docs/docs/user/references/editor-integration.md

```diff
@@ -20,3 +20,6 @@ KPOps provides JSON schemas that enable autocompletion and validation for some o
       ./docs/resources/editor_integration/settings.json
       --8<--
     ```
+
+!!! bug
+    Currently there is a [bug](https://github.com/redhat-developer/vscode-yaml/issues/523) in VScode's [Yaml Language Support](https://github.com/redhat-developer/vscode-yaml/) extension that gives an error when having both local (path) and remote (URI) schemas that match a file.
```

Comment:
for which filename does this occur, `config.yaml` or `pipeline.yaml`?

Reply:
Any file I think. I encountered it while playing around with the schemas and found an existing issue. Our user shouldn't run into such problems, under the assumed usage.

Reply:
ah, so it only occurs when you point the language server to the schema locally and on GitHub?
In that case, if it's not relevant to the recommended usage, I would suggest to remove this section. If it's only relevant to us during development and testing of the schemas we'll jsut have to deal with it I guess.

Reply:
Yeah, I guess it is out of scope for our documentation